BladeportBinaryTreeManager.Web/Controllers/UserController.cs

```diff
@@ -25,5 +25,25 @@ public List<UserDTO> GetUserList()
             return _userBl.GetUserList();
         }
 
+        [Route("adduser")]
+        [HttpPost]
+        public void AddUser ([FromBody] UserDTO user)
+        {
+            _userBl.AddUser(user);
+        }
+
+        [Route("edituser")]
+        [HttpPost]
+        public void EditUser([FromBody] UserDTO user)
+        {
+            _userBl.EditUser(user);
+        }
+
+        [Route("deleteuser")]
+        [HttpPost]
+        public void DeleteUser([FromBody] UserDTO user)
+        {
+            _userBl.DeleteUser(user);
+        }
```

Comment:
Return Success/Failure value and convert to ASYNC

Reply:
e.g. public bool async AddUser ([FromBody] UserDTO user)
       {
               return await _userBl.AddUser(user)
       }